_data/learn.yml

```
@@ -7,3 +7,15 @@
 - name: Learn Jekyll
   link: https://learn.cloudcannon.com/jekyll/why-use-a-static-site-generator/
   description: "Introduction to the world of static site generators"
+- name: Rails Guides
+  link: https://guides.rubyonrails.org/getting_started.html
+  description: "Get started with creating a Rails application from scratch"
+- name: Hanami 2.0
+  link: https://guides.hanamirb.org/v2.0/introduction/getting-started/
```

Let's make this generic so that we don't have to change it again and again when Hanami version changes in future.

```suggestion
- name: Hanami
  link: https://guides.hanamirb.org/
```

---

Addressed!